tests/tape/test_tape.py

```diff
@@ -998,13 +998,14 @@ def test_depth_expansion(self):
             # Each PauliX will then be decomposed into PhaseShift, RX, PhaseShift.
             qml.BasisState(np.array([1, 1]), wires=[0, "a"])
 
-            with QuantumTape() as tape2:
+            with QuantumTape():
                 # will be decomposed into a RZ, RY, RZ
                 qml.Rot(0.543, 0.1, 0.4, wires=0)
 
             qml.CNOT(wires=[0, "a"])
             qml.RY(0.2, wires="a")
-            qml.probs(wires=0), qml.probs(wires="a")
+            qml.probs(wires=0)
+            qml.probs(wires="a")
 
         new_tape = tape.expand(depth=3)
```

Comment:
```suggestion
        new_tape = tape.expand(depth=2)
```

Reply:
I was more thinking to change the docstring, sorry for the added hassle